app/scripts/modules/amazon/src/loadBalancer/configure/network/NLBListeners.tsx

```diff
@@ -25,6 +25,85 @@ export interface INLBListenersState {
   oidcConfigs: IAuthenticateOidcActionConfig[];
 }
 
+export interface INLBCertificateSelectorProps {
+  availableCertificates: IALBListenerCertificate[];
+  certificates: { [accountId: number]: IAmazonCertificate[] };
+  formik: FormikProps<IAmazonNetworkLoadBalancerUpsertCommand>;
+  app?: Application;
+  certificateTypes: string[];
+}
+
+export class CertificateSelector extends React.Component<INLBCertificateSelectorProps> {
+  public render() {
+    return <CertificateSelectorContent {...this.props} />;
+  }
+}
```

Comment:
You don't need this component because all it is doing is rendering the other component.

Reply:
fixed this, thx